streams-activitypub-dist/Dockerfile

```diff
@@ -3,6 +3,8 @@ MAINTAINER dev@streams.apache.org
 LABEL Description="apache-streams-activitypub-dist"
 WORKDIR /
 RUN mkdir -p /etc/fuseki
+RUN mkdir -p /etc/rdf
+ADD target/classes/*.ttl /etc/rdf/
```

Comment:
certain static ontologies incl activitystreams2.ttl make sense to embed in the distributable and load directly upon initialization